app/core/db.py

```
@@ -0,0 +1,35 @@
+from typing import AsyncGenerator
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)